src/amo/components/RatingsByStar/styles.scss

```diff
@@ -2,23 +2,37 @@
 
 .RatingsByStar-graph {
   color: $grey-50;
-  display: grid;
   font-size: $font-size-default;
-  grid-gap: 12px;
-  grid-template-columns: max-content auto max-content;
-  // This controls whitespace and should also match the bar height.
   line-height: 16px;
+  width: 100%;
+  border-spacing: 0 8px;
```

Comment:
This is going to fail our linter. Properties need to be specified in alphabetical order. You can also see this in the Travis results.

Reply:
Yes, Thanks for that @bobsilverberg. I noticed it now only. I will update that.